tests/unit_tests/data_steward/cdr_cleaner/clean_cdr_test.py

```diff
@@ -0,0 +1,56 @@
+import mock
+import unittest
+from cdr_cleaner import clean_cdr
+from constants.cdr_cleaner import clean_cdr as cdr_consts
+
+
+class CleanCDRTest(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        print('**************************************************************')
+        print(cls.__name__)
+        print('**************************************************************')
+
+    def setUp(self):
+        self.project_id = 'test project'
+        self.dataset_id = 'test dataset'
+        self.sandbox_dataset_id = 'test sandbox'
+        self.dest_table = 'dest_table'
+
+        self.function_name = 'anonymous function'
+        self.model_name = 'test module'
+        self.line_no = 1
+
+        self.query_dict = {
+            cdr_consts.QUERY: 'query',
+            cdr_consts.DESTINATION_DATASET: self.dataset_id,
+            cdr_consts.DESTINATION_TABLE: self.dest_table
+        }
+
+        self.module_info_dict = {
+            cdr_consts.MODULE_NAME: self.model_name,
+            cdr_consts.FUNCTION_NAME: self.function_name,
+            cdr_consts.LINE_NO: self.line_no
+        }
+
+    @mock.patch('inspect.signature')
+    @mock.patch('inspect.getsourcelines')
+    @mock.patch('inspect.getmodule')
+    def test_add_module_info_decorator(self, mock_get_module,
+                                       mock_getsourcelines, mock_signature):
```

Comment:
@ChaoPang Codacy is complaining about the `mock_signature`.  It doesn't look like you're using it.  You should be able to remove it and its patch.

Reply:
@lrwb-aou thanks for pointing this out, issue addressed.